<launch>
	<arg name="agent_num" default="6" />
	
	<node pkg="sunray_swarm" name="swarm_leader_follower_rmtt" type="swarm_leader_follower" output="screen">
		<param name="agent_type" value="0" />
		<param name="agent_num" value="$(arg agent_num)" />
	</node>
</launch>